src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
Also @ericvicenti sounds like with `navigation` being passed, we can just remove `navigationState`, right?